cli/main.rs

```diff
@@ -1404,34 +1396,40 @@ pub fn main() {
   colors::enable_ansi(); // For Windows 10
 
   let args: Vec<String> = env::args().collect();
-  let standalone_res = match standalone::extract_standalone(args.clone()) {
-    Ok(Some((metadata, bundle))) => {
-      run_basic(standalone::run(bundle, metadata))
-    }
-    Ok(None) => Ok(()),
-    Err(err) => Err(err),
-  };
-  // TODO(bartlomieju): doesn't handle exit code set by the runtime properly
-  unwrap_or_exit(standalone_res);
-
-  let flags = match flags::flags_from_vec(args) {
-    Ok(flags) => flags,
-    Err(err @ clap::Error { .. })
-      if err.kind == clap::ErrorKind::DisplayHelp
-        || err.kind == clap::ErrorKind::DisplayVersion =>
-    {
-      err.print().unwrap();
-      std::process::exit(0);
+
+  let exit_code = async move {
```

Comment:
This seems like unrelated change. I suggest to revert

Reply:
@bartlomieju this was done as both `extract_standalone` and `standalone::run` have been converted to async functions.

I wrapped the calls to these and also the call to the async fn `get_subcommand`  in an async block so we call `run_basic` only once.